Library/Homebrew/formula_auditor.rb

```diff
@@ -289,6 +289,12 @@ def audit_deps
 
           next unless @core_tap
 
+          if dep_f.deprecated? && !formula.deprecated?
```

Comment:
```suggestion
          if dep_f.deprecated? && !formula.deprecated? && !formula.disabled?
```

Reply:
Wouldn't we want to know when formula are more than deprecated?

Reply:
What do you mean by "more than deprecated"?